engines/group/app/models/group/group.rb

```diff
@@ -6,11 +6,11 @@ class Group < ActiveRecord::Base
     has_many :memberships, dependent: :destroy
     has_many :users, through: :memberships
     has_many :admins,
-      -> { where "memberships.role = #{ Membership::ROLES[:admin] }" },
+      -> { where "memberships.role = #{Role.new(:admin).index}" },
     through: :memberships,
       source: :user
     has_many :waiters,
-      -> { where "memberships.role = #{ Membership::ROLES[:waiting] }" },
+      -> { where "memberships.role = #{Role.new(:waiters).index}" },
```

Comment:
I'm not sure if this role make sense. We have invitation row to track this status